src/libraries/System.Private.CoreLib/src/System/IO/FileStatus.OtherUnix.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+namespace System.IO
+{
+    internal partial struct FileStatus
```

Comment:
Per https://github.com/dotnet/runtime/pull/49555/files#r705034658, this becomes `FileStatus.[Creation]Time.Unix.cs` and contains the non-OSX Unix implementation.

Reply:
See https://github.com/dotnet/runtime/pull/49555#discussion_r705042466.